drivers/uktty/pl011/pl011.c

```diff
@@ -90,7 +90,7 @@ static void init_pl011(__u64 bas)
 
 	/* Mask all interrupts */
 	PL011_REG_WRITE(REG_UARTIMSC_OFFSET,
-		PL011_REG_READ(REG_UARTIMSC_OFFSET) & 0xf800);
+			PL011_REG_READ(REG_UARTIMSC_OFFSET) & 0xf800);
```

Comment:
Could we maybe have `drivers/uktty/pl011: Whitespace fixes for checkpatch` squashed into the migration commit? :thinking:

Reply:
It's also one possible way of many. I considered both that option as well as the possibility to do all checkpatch changes in one commit (or even in the same one as the migration), but in the end I concluded it would be cleaner to do this in discrete steps. If you have a strong opinion / reasoning about it I don't mind to change it though.

Reply:
> If you have a strong opinion / reasoning about it I don't mind to change it though.

Nah, my only concern is usually when it comes to bisecting and looking at git history to not be unnecessarily contaminated.

> I concluded it would be cleaner to do this in discrete steps

Ok I see. Sure, it's just one commit after all. Let's keep it like this if you say you considered it already and believe it to be cleaner: migration and checkpatch fixes. :+1: